lib/cogctl/actions/relays/enable.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Cogctl.Actions.Relays.Enable do
```

Comment:
I played with this a bit and it didn't really simplify much, I still need the case statement to say whether things were enabled or disabled. I also like enable and disable being 2 separate commands as opposed to a toggle. It's clear what is happening.